{"uuid": "99242910-e046-49da-9db5-52a3a78f9f0a", "children": ["d47b5484-91cd-44c4-96d9-42bb9743b096"], "befores": [{"name": "appium_driver", "status": "passed", "start": 1721327056847, "stop": 1721327068053}], "afters": [{"name": "appium_driver::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started\nStacktrace:\nNoSuchDriverError: A session is either terminated or not started\n    at asyncHandler (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\protocol.js:309:15)\n    at C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\protocol.js:512:15\n    at Layer.handle [as handle_request] (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:284:15\n    at param (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:365:14)\n    at param (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:376:14)\n    at Function.process_params (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:421:3)\n    at next (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at logger (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\morgan\\index.js:144:5)\n    at Layer.handle [as handle_request] (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:328:13)\n    at C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:286:9\n    at Function.process_params (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at jsonParser (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\body-parser\\lib\\types\\json.js:113:7)\n    at Layer.handle [as handle_request] (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:328:13)\n    at C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:286:9\n    at Function.process_params (C:\\Users\\HP\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\express\\lib\\router\\index.js:346:12)\n", "trace": "  File \"C:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"E:\\WorkSpace\\appium-python-pytest\\TestCases\\conftest.py\", line 37, in appium_driver\n    driver.quit()\n  File \"C:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 476, in quit\n    self.execute(Command.QUIT)\n  File \"C:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 125, in check_response\n    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\n"}, "start": 1721327208086, "stop": 1721327208096}], "start": 1721327056847, "stop": 1721327208097}